tests/test_redis_backend.py

```diff
@@ -126,3 +126,10 @@ def test_lookup(self):
             end=datetime.date(2014, 2, 5),
             buckets=['group1', 'group2'],
         ))
+
+    def test_lookup_no_values(self):
+        self.assertEqual([], self.backend.lookup(
+            ActivityTracker.PERIOD_MONTHLY,
+            start=datetime.date(2013, 10, 5),
+            end=datetime.date(2013, 2, 5),
```

Comment:
Not intentionally, but if the end date is `2014-02-05`, it wouldn't have uncovered the bug. A the very least, I'll make the start/end the same to avoid confusion